lua/telescope/_extensions/project.lua

```diff
@@ -63,15 +63,20 @@ local project = function(opts)
   check_for_project_dirs_file()
   local projects = {}
 
-  -- format for projects is title of project=~/this/path/name
   for line in io.lines(project_dirs_file) do
     local title, path = line:match("^(.-)=(.-)$")
+    local last_accessed = vim.loop.fs_stat(path).atime.sec
```

Comment:
`vim.loop.fs_stat` can return nil so accessing `atime` will result in an error.
Also `path` needs to be extended. So `~/repos/project` will result in nil. `vim.fn.expand('~/repos/project')` will make it into `/home/conni/repos/project`. Just mentioning i don't know what the project paths are right now.

Also can you be more specific whats not working here

Reply:
Thanks @Conni2461 useful comments. I just can't get the projects to display as a sorted list by access time. The order of the projects display differently with the changes, but they don't update when I access files within the various project directories.